Eatery/Views/Eateries/EateryScreenViews/ExpandedMenus/ExpandedMenuRow.swift

```diff
@@ -23,6 +39,10 @@ class ExpandedMenuRow: UIView {
     /// Constant used multiple times in codebase to get/set height of ExpandedMenuRows
     static let heightConst: CGFloat = 44
 
+    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
```

Comment:
Touches began might be a little unnatural. A normal tap gesture recognizer might be the move.